roles/cifmw_cephadm/tasks/post.yml

```diff
@@ -63,7 +63,15 @@
       vars:
         ceph_command: "radosgw-admin"
 
+    - name: Check if Ceph user exists
+      become: true
+      ansible.builtin.command:
+        cmd: >-
+          {{ cifmw_cephadm_ceph_cli }} user info --uid openstack
+      register: cifmw_ceph_s3_user
+
     - name: Ceph rgw - create access/secret key
+      when: "no user info saved in {{ cifmw_ceph_s3_user.stdout }}"
```

Comment:
It will be in `stderr` not `stdout`  and we also need an `ignore_errors: true` or similar so that if the new task fails (and it will fail the first time) then it does not stop the play with the error below.

```
    TASK [cifmw_cephadm : Check if Ceph user exists _raw_params={{ cifmw_cephadm_ceph_cli }} user info --uid openstack] ***
    Friday 15 November 2024  12:30:33 -0500 (0:00:00.080)       0:05:04.139 *******
    fatal: [compute-vqt0nogt-0]: FAILED! => changed=true
      cmd:
      - podman
      - run
      - --rm
      - --net=host
      - --ipc=host
      - --volume
      - /etc/ceph:/etc/ceph:z
      - --volume
      - /home/ceph-admin/assimilate_ceph.conf:/home/assimilate_ceph.conf:z
      - --entrypoint
      - radosgw-admin
      - registry-proxy.engineering.redhat.com/rh-osbs/rhceph:7-453
      - --fsid
      - 1a1b33b2-e12d-5489-9646-51bfade45372
      - -c
      - /etc/ceph/ceph.conf
      - -k
      - /etc/ceph/ceph.client.admin.keyring
      - user
      - info
      - --uid
      - openstack
      delta: '0:00:09.977484'
      end: '2024-11-15 17:30:43.756108'
      msg: non-zero return code
      rc: 22
      start: '2024-11-15 17:30:33.778624'
      stderr: 'could not fetch user info: no user info saved'
      stderr_lines: <omitted>
      stdout: ''
      stdout_lines: <omitted>
```

Reply:
ok, I think we should probably switch to `ansible.builtin.shell` instead of using `ansible.builtin.command`